siriuspy/siriuspy/sofb/csdev.py

```diff
@@ -804,6 +847,32 @@ def get_sofb_database(self, prefix=''):
         dbase.update(self._add_prefix(db_ring, prefix))
         return dbase
 
+    def get_corrs_database(self, prefix=''):
+        """Return OpticsCorr-Chrom Soft IOC database."""
```

Comment:
wouldn't it be `"Return SOFB correctors database."` ?

Reply:
right, done!